rmgpy/tools/canteramodel.py

```diff
@@ -167,25 +177,28 @@ def generate_cantera_conditions(reactor_type_list, reaction_time_list, mol_frac_
         for reactor_type in reactor_type_list:
             for reaction_time in reaction_time_list:
                 for mol_frac in mol_frac_list:
-                    for P in Plist:
-                        for V in Vlist:
-                            conditions.append(CanteraCondition(reactor_type, reaction_time, mol_frac, P0=P, V0=V))
+                    for surface_mol_frac in surface_mol_frac_list:
+                        for P in Plist:
+                            for V in Vlist:
+                                conditions.append(CanteraCondition(reactor_type, reaction_time, mol_frac, surface_mol_frac=surface_mol_frac, P0=P, V0=V))
```

Comment:
See above comment - is this going to result in the CanteraCondition getting None for the surface_mol_frac?

Reply:
I've tested the new version with the empty list and it works for both surface and non-surface mechanisms